### PR TITLE
javascript/javascript3/week3

### PR DESCRIPTION
**Pull request by Rasim-Deniz**

### CODE REVIEW

File: javascript/javascript3/week3/paint.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            context.arc(this.x, this.y, this.r, 0, 2 * Math.PI);
+            context.stroke();
+        })
+
+    }
+}
+const c1 = new Circle(50, 50, 20, 0, 2 * Math.PI, "#000000");
+c1.draw();
+c1.cursorCircle();
+c1.randomCircle();
```
Comment (MirSzabo) [inline javascript/javascript3/week3/paint.js:52]: looks very nice 
good job 👍

File: javascript/javascript3/week3/promise.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+// const classMate8 = fetchUser("seenachacko");
+// const classMate9 = fetchUser("Bhuvaneshwari29");
+// const classMate10 = fetchUser("mahakarzoun");
+// const classMate11 = fetchUser("DarinkaGM");
+// const classMate13 = fetchUser("adnan832");
+// const classMate14 = fetchUser("shanhaider23");
+// const classMate15 = fetchUser("WENCHO22");
+// const classMate16 = fetchUser("Hani-far");
+
+Promise.all([classMate1, classMate2, classMate3]);
```
Comment (MirSzabo) [inline javascript/javascript3/week3/promise.js]: use chaining to catch errors

File: javascript/javascript3/week3/shopping-cart.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                break;
+            case "TRY":
+                this.price *= 1.27
+                break;
+            case "ZAR":
+                this.price *= 2.37
+        }
+        return this.price.toFixed(2);
+    }
+}
```
Comment (MirSzabo) [inline javascript/javascript3/week3/shopping-cart.js:111]: lot of work with currency exchange! :) 
did you look on some APIs you can use for accurate exchange rates?

**Review by MirSzabo — COMMENTED** [review]
nice work!
just few comments to look at 👍

File: javascript/javascript3/week3/paint.js (hunk 1 above)
Comment (Rasim-Deniz) [reply javascript/javascript3/week3/paint.js:52]: Thanks a lot for your time.

File: javascript/javascript3/week3/shopping-cart.js (hunk 3 above)
Comment (Rasim-Deniz) [reply javascript/javascript3/week3/shopping-cart.js:111]: Yes, but got some errors with it.

**Pull request merged**